### DISCUSSION TITLE
Add Beca BAC-005 fancoil thermostat to supported devices

### DISCUSSION THREAD

**Discussion by PsychoRS** [make-all/tuya-local#1465] [General]
Hi,

Becasmart BAC-005 can be added to the DEVICES.md as a supported thermostat, tested by me and working flawlessly. In fact, I'm pretty sure that the BAC series (005, 006, 002, 003...) are all the same internal devices, with only cosmetic differences. All of them might work with the BAC-002 configuration.

**Comment by make-all** [comment]
Do you have the  product_id for this device?

**Reply by PsychoRS** [reply]
Of course:

`"product_id": "N8bUqOZ8HBQjU0K0"`
